src/Components/ItemPropertiesDrawer.jsx

```diff
@@ -43,16 +41,16 @@ export default function ItemPropertiesDrawer({
 }
 
 
-const useStyles = makeStyles((props) => ({
+const useStyles = makeStyles((props) => (preprocessMediaQuery(MOBILE_WIDTH, {
```

Comment:
this is interesting - I don't fully understand what is going on here - but looks cool :)
it recursively steps through all of the style objects and makes mobile width available to each object?

Reply:
Yeah, you just type in MOBILE_WIDTH instead of e.g. 900px in some places and 500px in others.  We can use this elsewhere but didn't want to pull in a bunch of files in this CL.